src/components/checkout/checkout.module.scss

```diff
@@ -0,0 +1,18 @@
+@import '@styles/theme.module.scss';
+
+.root {
+    margin: 10px;
+}
+.button {
```

Comment:
do we really need a new button? why not use a generic one from common-styles.module.scss?